crates/fuel-core/src/graphql_api/storage.rs

```diff
@@ -19,18 +18,16 @@ pub mod transactions;
 pub enum Column {
     /// The column id of metadata about the blockchain
     Metadata = 0,
-    /// See [`Receipts`](receipts::Receipts)
-    Receipts = 1,
     /// The column of the table that stores `true` if `owner` owns `Coin` with `coin_id`
```

Comment:
Hmmm I'm not sure if it actually makes sense to reassign all of the numbers instead just leaving a gap here